uptane-standard.md

```diff
@@ -757,7 +767,7 @@ The filename used to identify the latest known image (i.e., the file to request
 1. Load the Targets metadata file from the Director repository.
 2. Find the Targets metadata associated with this ECU identifier.
 3. Construct the Image filename using the rule in {{metadata_filename_rules}}, or use the download URL specified in the Director metadata.
-4. If there is no Targets metadata about this image, abort the update cycle and report that there is no such image. Otherwise, download the image (up to the number of bytes specified in the Targets metadata), and verify that its hashes match the Targets metadata.
+4. If there is no Targets metadata about this image, abort the update cycle and report that there is no such image. Additionally, in the case of failure, the ECU SHALL overwrite its Targets metadata file with its previous Targets metadata file. Otherwise, download the image (up to the number of bytes specified in the Targets metadata), and verify that its hashes match the Targets metadata.
```

Comment:
Seems oddly specific here. Shouldn't this generally be the case for any metadata verification failure at all? @JustinCappos @awwad That is, no new metadata should replace previous metadata until all of them are verified.

Also, we should not mention files directly, as not all implementors would use files.

Reply:
```suggestion
4. If there is no Targets metadata about this image, abort the update cycle and report that there is no such image. Additionally, in the case of failure, the ECU SHALL retain its previous Targets metadata instead of using the new Targets metadata. Otherwise, download the image (up to the number of bytes specified in the Targets metadata), and verify that its hashes match the Targets metadata.
```